ipl/main.c

```diff
@@ -1940,6 +1950,12 @@ int fix_attributes(char *path, u32 *total)
 			memcpy(&path[i+1], fno.fname, k + 1);
 			path[i + k + 2] = 0;
 
+			if (is_root && !strcmp(path, "Nintendo"))
```

Comment:
This will always fail.
The cmp will compare "//Nintendo" with "Nintendo".

Move it above `memcpy(&path[i], "/", 1);` to save time from the for loop and memcpy.

And change the path to `fno.fname`
This way the cmp will compare "Nintendo" with "Nintendo" if is_root.

Reply:
Yea I figured that out pretty quickly. I actually corrupted my Nintendo dir in my testing. =(